Documentation/Blazorise.Docs/Pages/Docs/Extensions/DataGrid/DataGridApi.razor

```diff
@@ -480,18 +480,78 @@
     <DocsAttributesItem Name="ReverseSorting" Type="bool" Default="false">
         Whether the sort direction will be reversed.
     </DocsAttributesItem>
-    <DocsAttributesItem Name="TextAlignment" Type="TextAlignment?" Default="Default">
+    <DocsAttributesItem Name="TextAlignment" Type="TextAlignment" Default="Default">
         Defines the alignment for display cell.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="TextTransform" Type="TextTransform" Default="Default">
+        Gets or sets the text transformation for display cell.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="TextDecoration" Type="TextDecoration" Default="Default">
+        Gets or sets the text decoration for display cell.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="TextWeight" Type="TextWeight" Default="Default">
+        Gets or sets the text weight for display cell.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="TextOverflow" Type="TextOverflow" Default="Default">
+        Determines how the text will behave when it is larger than a parent container for display cell.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="TextSize" Type="IFluentTextSize" Default="null">
+        Determines the font size of an element for display cell.
+    </DocsAttributesItem>
     <DocsAttributesItem Name="HeaderTextAlignment" Type="TextAlignment?" Default="null">
         Defines the alignment for column header cell.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="HeaderTextTransform" Type="TextTransform?" Default="null">
```

Comment:
Have we always been using nullable types in the docs?

Reply:
Do you mean the `?` sign? I believe so, we did where it was valid to be mentioned.